### DISCUSSION TITLE
is a ui like this possible with iced (or even rust)?

### DISCUSSION THREAD

**Discussion by AliBasicCoder** [hecrj/iced#1323] [Q&A]
https://user-images.githubusercontent.com/48116123/166126334-0cff7934-7054-45c4-965d-47abae810ad6.mp4

**Comment by 13r0ck** [comment]
Yes, buttons are create-able at runtime in iced. Iced doesn't use svelte's architecture though, so the code would be written differently, but the same outcome is possible

**Reply by AliBasicCoder** [reply]
Ok, but how to make this?

**Comment by unreal79** [comment]
Take a look at [download_progress](https://github.com/iced-rs/iced/tree/master/examples/download_progress) example.
